clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -304,7 +304,8 @@
     PREFERRED_LEADER_NOT_AVAILABLE(80, "The preferred leader was not available",
             PreferredLeaderNotAvailableException::new),
     GROUP_MAX_SIZE_REACHED(81, "The consumer group has reached its max size.", GroupMaxSizeReachedException::new),
-    FENCED_INSTANCE_ID(82, "The coordinator reports a more recent member.id associated with the consumer's group.instance.id.",
+    FENCED_INSTANCE_ID(82, "The broker rejected this static consumer since " +
```

Comment:
Probably fine, but I guess this could be `FENCED_GROUP_INSTANCE_ID` to match the config name.

Reply:
Yea, I think it should be fine to make the error name concise.